it/projects/infrastrutture-digitali.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Infrastrutture digitali e cloud
+lang: it
+permalink: /it/projects/infrastrutture-digitali.htm
+layout: project
+ref: cloud
+parent_ref: projects
+parent_menu: projects
+people: [paolo-de-rosa]
+toc: true
+medium_tag: cloudpa
+forum_category: cloud
+forum_limit: 3
+timeline:
+twitter_tag: cloudpa
+tweetdeck_id: 913421951103897600
+---
+
+### In breve
+
+Le infrastrutture digitali ricoprono un ruolo vitale per un gran numero di attività che sono parte della nostra quotidianità. Da un punto di vista strategico sono equivalenti e devono essere pertanto considerate al pari di risorse quali le reti di telecomunicazione, le autostrade, le centrali elettriche o gli acquedotti e le risorse produttive in genere. Per lo Stato rappresentano sempre più l’ossatura portante del sistema di servizi che le Pubbliche Amministrazioni utilizzano ed erogano ai cittadini. Per questo le infrastrutture digitali della PA devono essere affidabili, sicure ed economicamente sostenibili.
+
+Nel corso degli anni, tuttavia, l’infrastruttura tecnologica della Pubblica Amministrazione è stata sviluppata e organizzata in maniera casuale, lasciando le decisioni all’iniziativa di ogni singola amministrazione, senza una visione d’insieme, un coordinamento, una pianificazione. Il risultato è una giungla di migliaia di piccoli centri di elaborazione dati (Ced).
+
+Questo scenario pone l’esigenza di un processo di razionalizzazione dei data center esistenti, attraverso la dismissione di quelli obsoleti o non sufficientemente sicuri e affidabili. Una razionalizzazione che, grazie alle forti economie di scala, consentirebbe:
+
+- una consistente riduzione della spesa pubblica; 
+- una maggiore sicurezza delle infrastrutture contro i rischi informatici;
+- un innalzamento della qualità;
+- una migliore efficienza energetica.
+
+Il [Piano Triennale per l’informatica nella pubblica amministrazione 2017–2019](https://pianotriennale-ict.italia.it/), nell’ambito di un più generale processo di trasformazione digitale del Paese, ha avviato un percorso dedicato alle infrastrutture digitali, gettando le basi:
+
+- per la razionalizzazione di tutte le strutture fisiche (*data center*) esistenti;
+- per l’introduzione di infrastrutture digitali più sicure e affidabili mediante le tecnologie di *cloud computing*.    
+
+![Infrastrutture digitali](/assets/img/infrastrutture_digitali.png)
+
+### Il ruolo del Team
+
+Per poter dismettere i data center obsoleti è necessario che i servizi erogati vengano migrati su altre infrastrutture. Con questo obiettivo il Team per la trasformazione digitale insieme ad Agid ha definito una strategia diversificata che prevede diverse tipologie di infrastrutture in funzione delle caratteristiche del servizio consumato o erogato dalle amministrazioni.
```

Comment:
`Agid` -> `AgID`

Reply:
Su questo l'anno scorso l'Agid ha mandato una circolare interna dicendo che hanno cambiato logo e che pertanto d'ora in avanti non è più giusto scrivere AgID, ma bisogna scrivere AGID tutto maiuscolo (v. https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/le-parole-della-pubblica-amministrazione/a.html)

Visto che non siamo tenuti a stare dietro a questi cambiamenti, da quest'anno nei post e le altre cose ufficiali del team scriviamo "Agid" con la sola prima "A" maiuscola, come da regola generale per tutti gli acronimi (v. https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/suggerimenti-di-scrittura/stile-di-scrittura.html?highlight=sigle)